app/src/sharedTest/java/org/oppia/app/topic/review/TopicReviewFragmentTest.kt

```diff
@@ -25,10 +32,37 @@ import javax.inject.Singleton
 @RunWith(AndroidJUnit4::class)
 class TopicReviewFragmentTest {
 
+  @get:Rule
+  var topicActivityTestRule: ActivityTestRule<TopicActivity> = ActivityTestRule(
+    TopicActivity::class.java, /* initialTouchMode= */ true, /* launchActivity= */ false
+  )
+
+  @Test
+  fun testTopicReviewFragment_loadFragment_displayReviewSkills_isSuccessfully() {
```

Comment:
Suggest: ``testTopicReviewFragment_loadFragment_displayReviewSkills_isSuccessful()`` or ``testTopicReviewFragment_loadFragment_displaysReviewSkillsSuccessfully()``. The way the current test name reads is slightly awkward grammatically.

Reply:
Corrected to `testTopicReviewFragment_loadFragment_displayReviewSkills_isSuccessful()`